src/ServiceContainer/Driver/PantherFactory.php

```diff
@@ -75,6 +77,10 @@ public function buildDriver(array $config)
             $config['manager_options']['capabilities'][ChromeOptions::CAPABILITY] = $chromeOptions;
         }
 
+        if (isset($config['options']['browser']) && isset($config['manager_options']['capabilities']) && PantherTestCase::SELENIUM === $config['options']['browser']) {
```

Comment:
It's possible we want to do this for any browser, not just Selenium, no? :thinking: